prisma/schema.prisma

```diff
@@ -0,0 +1,24 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model Story {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
```

Comment:
Do we need an updatedAt? Are we allowing people to edit? How does this work for anonymous posts?

Reply:
I think it's just prudent to have this here. To your point, wouldn't be for user updates, but if it was updated in the admin portal (in the future) or even just set to `approved` or whatever.